packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -431,6 +442,25 @@ export class StripeHelper {
     });
   }
 
+  /**
+   * Remove the PayPal Billing Agreement ID from a customer.
+   *
+   * @param customer
+   * @param agreementId
+   */
+  async removeCustomerPaypalAgreement(
+    uid: string,
+    customer: Stripe.Customer,
+    billingAgreementId: string
+  ): Promise<any[]> {
```

Comment:
I prefer to either let TS infer the return type or use actual types (`Promise<Promise<Stripe.Customer | number>[]>`).

The type is a Promise of an array of Promises since the function is `async`.  But should the array be in a `Promise.all`?  An `any[]` resolves immediately.